index.html

```diff
@@ -8,13 +8,25 @@ <h1>HomeWork</h1>
         <em><u>This page was created for homework</em></u>
         <sup><mark>Lesson3</sup></mark>
         <p>The first table will be created below</p>
-        <h2>&#x1F447;</h2>
+            <h2>&#x1F447;</h2>
         <ol>
             <li>Firt item</li>
             <li>Second item</li>
             <li>Third item</li>
             <li>Fouth item</li>
             <li>Fifth item</li>
         </ol>
+        <center><h1>HomeWork Completed</h1></center>
```

Comment:
should in another order, like this:
`<h1><center>HomeWork Completed</center></h1>`

Reply:
accepted